src/Symfony/Component/Mailer/Bridge/Ohmysmtp/README.md

```diff
@@ -0,0 +1,12 @@
+OhMySMTP Bridge
+===============
+
+Provides [OhMySMTP](https://ohmysmtp.com) integration for Symfony Mailer.
+
```

Comment:
Please add a DSN example, you can check a README from the Notifier bridges, thanks